src/zope/app/apidoc/browser/configure.zcml

```diff
@@ -4,13 +4,9 @@
     xmlns:zcml="http://namespaces.zope.org/zcml"
     i18n_domain="zope">
 
-  <!-- BBB 2006/02/18, to be removed after 12 months -->
-  <layer
```

Comment:
This directive no longer seems to exist anywhere. Previously [`zope.app.publisher.browser` used to provide it](https://github.com/zopefoundation/zope.app.publisher/blob/3.4/src/zope/app/publisher/browser/meta.zcml#L132) but that disappeared somewhere after 3.4. Even though there was a `zcml:condition` attribute, you can't load this file independently (interestingly, you *could* load this file from another file), which made testing difficult. zope.app.rotterdam has the same issue.